Themes/default/BoardIndex.template.php

```diff
@@ -213,7 +212,7 @@ function template_bi_redirect_stats($board)
 
 	echo '
 						<p>
-							', comma_format($board['posts']), ' ', $txt['redirects'], '
+							', $txt['redirects'], ': ', comma_format($board['posts']), '
 						</p>';
```

Comment:
Don't really like the : being used here.  Is this to try and internationalize it better?

Reply:
Yes. it's better for internationalize. To avoid adding many new variables in all languages, such as:
```
$txt['posts'] = array(1 => '1 post', n => 'n posts');
$txt['redirects'] = array(1 => '1 redirect', n => 'n redirects') etc.
```